docs/reference/sql/endpoints/rest.asciidoc

```diff
@@ -522,6 +522,44 @@ POST /_sql?format=txt
 [IMPORTANT]
 The recommended way of passing values to a query is with question mark placeholders, to avoid any attempts of hacking or SQL injection.
 
```

Comment:
In this file, at the top, I believe you need to add a link to "Use runtime fields". As it is now (and how it was initially created) a list of sections is present here (https://elasticsearch_71487.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/sql-rest.html) as well as in the right hand side column.